server/src/main/java/com/defold/extender/services/GradleService.java

```diff
@@ -198,7 +197,7 @@ private String execCommand(String command, File cwd) throws ExtenderException {
         return pe.getOutput();
     }
 
-    static public Map<String, String> parseDependencies(String log) {
```

Comment:
No need for it to be public or static

Reply:
That could be argued for several other functions as well. In which case they should probably be moved to some utility class with only public static stateless helper functions.

Reply:
For sure. That's basically how ExtenderUtil came about.